bin/test-sbt-1.0

```diff
@@ -4,5 +4,5 @@
 
 SCALA_VERSION="2.12.6"
 
-runSbt +publishLocal
+runSbt "+++ ${SCALA_VERSION} publishLocal"
```

Comment:
No need to cross-publish. Publishing to Scala 2.12 should be enough.

Reply:
Is this meant to be `++` or, actually, `+++`?

Reply:
`+++`.

Reply:
Keep in mind Lagom is still using sbt 0.13, so this is using a sbt-doge command. See also https://github.com/lagom/lagom/blob/master/bin/test-2.12 for another place where we are using `+++`.

Reply:
I'd forgotten about sbt-doge already 😁